project_stock/models/project_task.py

```diff
@@ -203,10 +205,12 @@ def action_toggle_stock_moves_is_locked(self):
         return True
 
     def action_done(self):
+        """We need to apply sudo to avoid error if the user does not have access to
```

Comment:
The same for this.

Reply:
The corresponding comment has already been applied in the correct place.